rmw/include/rmw/topic_endpoint_info.h

```diff
@@ -135,6 +138,40 @@ rmw_topic_endpoint_info_set_topic_type(
   const char * topic_type,
   rcutils_allocator_t * allocator);
 
+/// Set the topic type hash in the given topic endpoint info data structure.
+/**
+ * This functions copies value `topic_type_hash` into data structure `topic_type_hash` member.
```

Comment:
```suggestion
 * This function copies value `topic_type_hash` into data structure `topic_type_hash` member.
```

But actually, I don't actually understand what this is trying to tell me.  I guess it copies the data from the given `rosidl_type_hash_t` structure into the `rmw_topic_endpoint_info_t` structure member, correct?  It might be better to rewrite this sentence to say that.

Reply:
Yeah sorry, copypasta result, I hadn't read it too closely, just changed out the names. I've done a pass to clean up the language on all the docstrings in this file.